Alignment/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
 project(Alignment)
+include(CTest)
 cmake_minimum_required(VERSION 2.8)
 aux_source_directory(. SRC_LIST)
-set(SOURCE_FILES main.cpp affinealignobj.cpp alignment.cpp affinealignment.cpp)
-add_executable(${PROJECT_NAME} ${SRC_LIST})
+set (CMAKE_CXX_STANDARD 11)
+set(SOURCE_FILES affinealignobj.cpp alignment.cpp affinealignment.cpp)
+
+add_executable(${PROJECT_NAME} main.cpp ${SOURCE_FILES})
```

Comment:
This change will allow getting executables of main.cpp file, which I was missing before. right?
Is it a good idea to have executables of all files? like the way, we have tests for all files (next 2 lines).